### PR TITLE
Refresh the PR tree when PR is approved

### LINKED ISSUE #1749: Consider refreshing the PR view when a pull request is approved
If I'm going through the "Waiting for my review" query and approving PRs, it would be nice if as I approve them they vanish from that query (which they will do when I click the button to refresh the view).

### PR DESCRIPTION
**Pull request by rishab-pdx**
Fixes #1749 
Made a call to ```pr.refreshList``` so that PR tree is updated.

### CODE REVIEW

File: src/github/pullRequestOverview.ts [hunk 1]
```diff
@@ -585,6 +585,8 @@ export class PullRequestOverviewPanel extends IssueOverviewPanel {
 				review: review,
 				reviewers: this._existingReviewers
 			});
+			//just making sure that pr review status is approved on git servers
```
Comment (RMacfarlane) [inline src/github/pullRequestOverview.ts:588]: I think this timeout isn't necessary, I believe GitHub will return the correct information immediately. Let me know if you've seen different behavior

Comment (rishab-pdx) [reply src/github/pullRequestOverview.ts:588]: When I was testing I could see some inconsistency with refreshing the list/view.  After I added ```timeout``` there wasn't in any inconsistency.

Comment (rishab-pdx) [reply src/github/pullRequestOverview.ts:588]: Let me know if you want me to remove the ```timeout```

**Comment by rishab-pdx** [comment]
> Thank you, this looks good! I think the timeout can be removed


When I was testing I could see some inconsistency with refreshing the list/view. After I added timeout there wasn't in any inconsistency. Let me know if you want me to remove timeout

File: src/github/pullRequestOverview.ts (hunk 1 above)
Comment (rishab-pdx) [reply src/github/pullRequestOverview.ts]: I have removed the ```timeout```.

**Pull request merged**